src/borg/archiver.py

```diff
@@ -2272,18 +2272,34 @@ def do_break_lock(self, args, repository):
 
                 # "sh:" pattern style is the default, so the following line is not needed:
                 P sh
-                R /
-                # can be rebuild
-                - /home/*/.cache
-                # they're downloads for a reason
-                - /home/*/Downloads
-                # susan is a nice person
-                # include susans home
-                + /home/susan
-                # don't backup the other home directories
-                - /home/*
-                # don't even look in /proc
-                ! /proc\n\n''')
+                # borg will consider only the files within a recursion root
+                # defined in this way: "R path/to/recursion_root"
+                R /home
+
+                # this won't be included because nowhere in the file
+                # there isn't a line like "R /" or "R /proc"
+                + /proc
+
+                # dir won't be included because of '!'
+                + /home/*/Downloads/dir
+                ! /home/*/Downloads
+
+                # from now on "pp:" pattern style will be applied
+                P pp
+
+                # This is correct
+                # the position of this line in the file is irrelevant
+                R /home/*/Pictures
```

Comment:
yes, because you just define recursion roots like that.

it is similar to calling borg like `borg create repo::archive root1 root2 root3`

Reply:
I know, I thought it was better to explicitly say that adding those lines